keepassxc-browser/content/passkeys-utils.js

```diff
@@ -16,65 +20,61 @@ const arrayBufferToBase64 = function(buf) {
     return window.btoa(str).replaceAll('+', '-').replaceAll('/', '_').replaceAll('=', '');
 };
 
-// Error checks for both registration and authentication
-const checkErrors = function(pkOptions) {
-    if (pkOptions.sameOriginWithAncestors !== undefined && pkOptions.sameOriginWithAncestors === false) {
-        throw new DOMException('Cross-origin register is not allowed.', DOMException.NotAllowedError);
+const checkErrors = function(pkOptions, sameOriginWithAncestors) {
+    if (!pkOptions) {
+        throw new Error('No publicKey configuration options were provided');
+    }
+
+    if (pkOptions.signal && pkOptions.signal.aborted) {
+        throw new DOMException('Abort signalled', DOMException.AbortError);
+    }
+
+    if (!sameOriginWithAncestors || (sameOriginWithAncestors && sameOriginWithAncestors === false)) {
```

Comment:
`sameOriginWithAncestors && sameOriginWithAncestors === false` is always `false` so I think `if (!sameOriginWithAncestors)` is enough.